andela/ah-centauri-frontend
# #164829177 search functionality

#### What does this PR do?
- Adds search functionality

#### Description of Tasks to be completed?
- Add search filters
- Add search button
- Add tests

#### How should this be manually tested?
- Clone the project: git clone https://github.com/andela/ah-centauri-frontend
- cd into the project folder
- checkout to branch `git checkout feature/164829177-search-functionality`
- Install dependencies: npm install
- Launch development server: npm start
- Click on the search icon on the homepage a modal will appear
- Search for your desired article

#### Any background context you want to provide?

#### What are the relevant Pivotal Tracker Stories?
[#164829177](https://www.pivotaltracker.com/n/projects/2321088/stories/164829177)

#### Screenshots(if appropriate)
<img width="1317" alt="Screenshot 2019-05-24 at 09 43 08" src="https://user-images.githubusercontent.com/37370926/58307867-a08dda80-7e08-11e9-881f-ecca902ad221.png">
<img width="1406" alt="Screenshot 2019-05-24 at 09 42 38" src="https://user-images.githubusercontent.com/37370926/58307873-a683bb80-7e08-11e9-87cc-8c9e8907ec1a.png">
